docs/quickstart.md

```
@@ -1,42 +1,52 @@
-2. Open a terminal in the Agent Zero directory and activate your conda environment (if you're using one).
-3. Run the following command:
+## Launching the Web UI (Docker)
+1. Pull the latest image:
 
 ```bash
-python run_ui.py
+docker pull agent0ai/agent-zero:latest
 ```
 
-4.  A message similar to this will appear in your terminal, indicating the Web UI is running:
+2. Run the container and let Docker assign a port:
```

markdownlint-cli2 (0.18.1)

Ordered list item prefix
Expected: 1; Actual: 2; Style: 1/1/1

(MD029, ol-prefix)